src/main/java/com/iridium/iridiumskyblock/commands/RegenCommand.java

```diff
@@ -47,6 +47,12 @@ public boolean execute(User user, Island island, String[] args, IridiumTeams<Isl
 
         Schematics.SchematicConfig schematicConfig = IridiumSkyblock.getInstance().getSchematics().schematics.get(schematic.get());
 
+        if(schematicConfig.regenCost.money != 0 && !schematicConfig.regenCost.bankItems.isEmpty()) {
```

Comment:
```suggestion
        if(schematicConfig.regenCost.money != 0 || !schematicConfig.regenCost.bankItems.isEmpty()) {
```

I think this should be or no?

Reply:
for example if the vault money cost is 0, but I say it costs 5 island crystals, this should still be valid?

Or If I said it was 10,000 but 0 bank items, thats also valid, no?